packages/wrangler/templates/d1-beta-facade.js

```diff
@@ -238,4 +238,11 @@ var shim_default = {
 		return worker.trace(traces, getMaskedEnv(env), ctx);
 	},
 };
-export { shim_default as default };
+var maskDurableObjectDefinition = (cls) =>
+	class extends cls {
+		constructor(state, env) {
+			super(state, getMaskedEnv(env));
+		}
+	};
+__DO_REEXPORTS__;
```

Comment:
If there are no durable objects to export this would end up leaving an empty statement here, right?

How about instead, we remove the semicolon here, and ensure that any injected exports always end with a semicolon?

```suggestion
__DO_REEXPORTS__
```

Reply:
Can the source for CFSQL/d1js be included here rather than a build output file?

Reply:
Actually no @petebacondarwin - looks like esbuild strips out the stray `;`